docs/reference/ml/apis/delete-dfanalytics.asciidoc

```diff
@@ -0,0 +1,52 @@
+[role="xpack"]
+[testenv="platinum"]
+[[delete-dfanalytics]]
+=== Delete {dfanalytics-jobs} API
```

Comment:
I think we should make the same terminology changes in the matching Java HLRC pages (e.g. https://www.elastic.co/guide/en/elasticsearch/client/java-rest/master/java-rest-high-x-pack-ml-delete-data-frame-analytics.html) . That is to say, it should have the "job" terminology too.  We can do this is a follow-up PR if necessary.

Reply:
I created a note about it on the Stack docs kanban board.
https://github.com/orgs/elastic/projects/172#card-23571524